src/block_misc.rs

```diff
@@ -1067,11 +1067,11 @@ mod tests {
 
     #[test]
     fn test_join_1_block() {
-        let mut mi = model_instance::ModelInstance::new_empty().unwrap();
+        let mi = model_instance::ModelInstance::new_empty().unwrap();
         let mut bg = BlockGraph::new();
         let input_block_1 = block_misc::new_const_block(&mut bg, vec![2.0, 3.0]).unwrap();
         let join_block = new_join_block(&mut bg, vec![input_block_1]).unwrap();
-        let observe_block =
+        let _observe_block =
```

Comment:
can't the operation simply be executed without the assignment, rather than adding the underscore prefix?

Reply:
You mean like `let _  = ...`? That will cause the returned value to be Droped at the end of the statement instead of at the end of the scope because there is no binding to hold it.